src/scenes/VapaaSivistystyo/components/LupaHistoryItem.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import {useIntl} from "react-intl";
+import moment from 'moment';
+import styled from "styled-components";
+import Media from "react-media";
+
+import { MEDIA_QUERIES } from "../../../modules/styles";
+
+import { API_BASE_URL } from "../../../modules/constants";
+import { Td, Tr } from "../../../modules/Table";
+import common from "../../../i18n/definitions/common";
+
+const LupaText = styled.span`
+  margin: 10px;
+
+  @media ${MEDIA_QUERIES.MOBILE} {
+    display: flex;
+    flex-direction: column;
+  }
+`;
+
+const TextPartial = styled.span`
+  margin-right: 10px;
```

Comment:
Same here. TextPartial should be removed. Only one TailwindCSS's style class is needed.